library/alloc/src/vec/mod.rs

```diff
@@ -1074,10 +1076,10 @@ impl<T, A: Allocator> Vec<T, A> {
 
     /// Converts the vector into [`Box<[T]>`][owned slice].
     ///
-    /// If the vector has excess capacity, its items will be moved into a
-    /// newly-allocated buffer with exactly the right capacity.
+    /// Before doing the conversion, this method calls [`shrink_to_fit`] to remove excess capacity.
```

Comment:
Rather that documenting how it's implementing (calling a specific method) it should describe behavior. "discards excess capacity, like shrink_to_fit" or something like that.